hyperlink/_url.py

```diff
@@ -861,6 +917,8 @@ def replace(self, scheme=_UNSET, host=_UNSET, path=_UNSET, query=_UNSET,
             port=_optional(port, self.port),
             rooted=_optional(rooted, self.rooted),
             userinfo=_optional(userinfo, self.userinfo),
+            family=_optional(family, self.family),
+            uses_netloc=_optional(uses_netloc, self.uses_netloc)
```

Comment:
Functionally, `uses_netloc` is there because for unrecognized schemes parsed using `from_text`, URL will persist whether or not the original URL used the netloc slashes.

Logistically, I can't remember if I intended to include it in this PR or if I just saw the big regression of not passing through `family` and `uses_netloc` -- both in the constructor and the docstring of `replace`, but missing in the function signature itself -- and fixed it post haste.